syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.github.alandiegosantos.monorepo.proto.rating";
option java_outer_classname = "RatingProto";
option objc_class_prefix = "GTR";

package rating;

message RatingRequest {
    string reviewId = 1;
}

message Rating {
    string reviewId = 1;
    int32 rating = 2;
}

service RatingService {
  rpc GetRating (RatingRequest) returns (Rating) {}
}
